em_workflows/czi/flow.py

```diff
@@ -1,28 +1,85 @@
 from pathlib import Path
+from typing import List, Dict
+import SimpleITK as sitk
 from prefect import Flow, Parameter, task
+from pytools.HedwigZarrImage import HedwigZarrImage
+from pytools.HedwigZarrImages import HedwigZarrImages
 from em_workflows.file_path import FilePath
 from em_workflows.utils import utils
-from em_workflows.config import Config
 from prefect.run_configs import LocalRun
 from .constants import VALID_CZI_INPUTS
+from .config import CZIConfig
+
+
+def rechunk_zarr(zarr_fp: Path) -> None:
+    images = HedwigZarrImages(zarr_fp, read_only=False)
+    for _, image in images.series():
+        image.rechunk(512)  # config.CHUNK_SIZE
+
+
+def gen_thumb(image: HedwigZarrImage, file_path: FilePath) -> dict:
```

Comment:
I believe `gen_thumb` should also take `image_name` into account. Because right now, all the assets are being saved to the same filename. @philipmac